alert-policies/activemq/TotalConnections.yml

```diff
@@ -0,0 +1,27 @@
+name: Total Connections
+
+description: |+
+  This alert is triggered when the Total connections is less than 1 during 5 minutes.
```

Comment:
please check this grammar mistake.
OLD:
This alert is triggered when the Total connections is less than 1 during 5 minutes.
Update needed:
This alert is triggered when the total connections are less than 1 during 5 minutes.